### PR TITLE
BREAKING CHANGE: Add component prefix to CSS variables #DS-1105 

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Review by literat — APPROVED** [review]
Nice 👍 I looks like it should be working :-)

**Review by curdaj — APPROVED** [review]

File: docs/migrations/web/MIGRATION-v2.md [hunk 1]
```diff
@@ -19,6 +20,22 @@ Introducing version 2 of the _spirit-web_ package
 
 ## General Changes
 
+### CSS Variables Component Prefix
+
+All CSS variables in the components now have a component name prefix.
+
+- In Stack the `--gap` is now `--stack-gap`
+- In helper `.text-truncate-multiline` the `--lines` is now `--text-truncate-lines`
+- The gradient `--angle` is now `--gradient-angle`
```
Comment (adamkudrna) [inline docs/migrations/web/MIGRATION-v2.md:29]:
```suggestion
- In Stack, `--gap` is now `--stack-gap`.
- In the `.text-truncate-multiline` helper, `--lines` is now `--text-truncate-lines`.
- The gradient `--angle` is now `--gradient-angle`.
```

At least that's my poor 🇬🇧 feeling 🤷🏻

File: packages/web/src/scss/helpers/text/_text.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
 .text-truncate-multiline {
     // stylelint-disable-next-line value-no-vendor-prefix -- We need the old flexbox syntax to make the trick work.
     display: -webkit-box;
-    -webkit-line-clamp: var(--lines, 3);
+    -webkit-line-clamp: var(--text-truncate-lines, 3);
```
Comment (adamkudrna) [inline packages/web/src/scss/helpers/text/_text.scss:11]: No demo edits? At least this one must be found in a README or demo…

**Review by adamkudrna — COMMENTED** [review]
I think READMEs and demos should be updated too (didn't check though).

**Review by pavelklibani — APPROVED** [review]

File: packages/web/src/scss/helpers/text/_text.scss (hunk 2 above)
Comment (crishpeen) [reply packages/web/src/scss/helpers/text/_text.scss:11]: nope, it is nowhere documented 😨

**Pull request merged**